docs/_js/ErrorCodesComponent.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+/* global React ReactDOM errorMap:true */
+'use strict';
+
+function replaceArgs(msg, argList) {
+  let argIdx = 0;
+  return msg.replace(/%s/g, function() {
+    const arg = argList[argIdx++];
+    return arg === undefined ? '[missing argument]' : arg;
+  });
+}
+
+function segmentify(str) {
+  const urlRegex = /(https:\/\/fb\.me\/[a-z\-]+)/g;
+  const matchResult = str.match(urlRegex);
+  if (!matchResult) {
+    return str;
+  }
+
+  const segments = str.split(urlRegex);
+
+  for (let i = 0; i < segments.length; i++) {
+    const matchIdx = matchResult.indexOf(segments[i]);
```

Comment:
You shouldn't need to run a separate match; with the split regex you have the URLs will always be in odd positions and the text in even positions.